src/LogProviders/IBackendBuildTableDataSource.cs

```diff
@@ -1,14 +1,12 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.Build.Framework;
-using Microsoft.VisualStudio.Shell.TableManager;
+//using Microsoft.VisualStudio.Shell.TableManager;
```

Comment:
Did you comment this out by mistake?

Reply:
That import showed up as unused because I had removed the ITableDataSource interface from this file
I'm not super sure if I can remove the ITableDataSource interface without breaking other code, although I haven't seen any problems related to it yet.